up-transport-vsomeip/src/message_conversions.rs

```diff
@@ -269,6 +259,29 @@ where {
 
         Ok(vsomeip_msg)
     }
+
+    fn ucode_to_return_code_e(ucode: UCode) -> vsomeip::return_code_e {
```

Comment:
Apologies @dmacattack -- I should have posted [this portion](https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust/pull/22) of the SOME/IP spec in task #20.